test/code_block_fenced.md

```diff
@@ -9,9 +9,16 @@ And here is more text
 This is a code block that won't trigger.
 ```
 
+```text
+This is code
+ 
+with an empty line.
```

Comment:
Can this say “with a line with trailing spaces”?

Reply:
👍